tests/cases/debugger_hlsl_line_stepping_no_inline_param.amber

```diff
@@ -114,57 +112,46 @@ VS_OUTPUT main(float4 pos : POSITION,
          %53 = OpExtInst %void %1 DebugLocalVariable %14 %43 %41 8 13 %52 FlagIsLocal
          %44 = OpExtInst %void %1 DebugTypeMember %12 %47 %41 2 3 %43 %uint_0 %uint_128 FlagIsProtected|FlagIsPrivate
          %45 = OpExtInst %void %1 DebugTypeMember %13 %47 %41 3 3 %43 %uint_128 %uint_128 FlagIsProtected|FlagIsPrivate
-               OpLine %7 6 1
```

Comment:
So as discussed, `OpLine` is treated as a "this-is-a-steppable-line". It would be great if these were only emitted for statements.